lib/lure/command/branchsync.go

```diff
@@ -0,0 +1,67 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/coveooss/lure/lib/lure/project"
+
+	"github.com/coveooss/lure/lib/lure/log"
+)
+
+func SynchronizedBranchesCommand(project project.Project, sourceControl sourceControl, provider provider, args map[string]string) error {
+	fromBranch, ok := args["from"]
+	if !ok {
+		return errors.New("Missing argument 'from'")
+	}
+	toBranch, ok := args["to"]
+	if !ok {
+		return errors.New("Missing argument 'to'")
+	}
+
+	return synchronizedBranches(project, sourceControl, provider, fromBranch, toBranch)
+}
+
+func synchronizedBranches(project project.Project, sourceControl sourceControl, provider provider, fromBranch string, toBranch string) error {
+
+	// if git, setup a tracking branch to avoid having to prefix origin/ later
+	if _, err := sourceControl.Update(toBranch); err != nil {
+		return err
+	}
+
+	if _, err := sourceControl.Update(fromBranch); err != nil {
+		return err
+	}
+
+	commits, err := sourceControl.LogCommitsBetween(toBranch, fromBranch)
```

Comment:
`LogCommitsBetween` 🤔  why "Log" here, it is returning commits. 

So should it be called `GetCommitsBetween` or just `CommitsBetween` ?